otaclient/app/create_standby/legacy_mode.py

```diff
@@ -306,7 +313,9 @@ def _create_regular_file(self, reginf: RegularInf):
         # TODO(20221017): should we report the real downloaded size(if compression enabled),
         #                 or keep the previous behavior that report the original file size?
         processed.size = dst.stat().st_size
-        processed.elapsed_ns = time.thread_time_ns() - begin_time
+        # NOTE: for download operation, the download time is already recorded
+        if processed.op != RegProcessOperation.OP_DOWNLOAD:
+            processed.elapsed_ns = time.thread_time_ns() - begin_time
```

Comment:
What do you think this code? default download_time is set to zero. This code can remove `if processed.op != RegProcessOperation.OP_DOWNLOAD:` statement.
```suggestion
        processed.elapsed_ns = time.thread_time_ns() - begin_time + download_time
```

Reply:
download_time is unbounded if op is not Download though.

Reply:
In the beginning of this function, add
```
dowlonad_time = 0
```
and remove L264 and L307

Reply:
Emmm I think it is not so neccessary to do so. And download_time for other op is strange I think.

Reply:
The background to this comment is that the assignment of processed.elapsed_ns exists in the multiple lines and this if-statement is difficult to understand.
I think when the OP is link or copy, `download_time = 0` is not strange.